datastore/save_test.go

```diff
@@ -321,6 +322,27 @@ func TestSaveFieldsWithInterface(t *testing.T) {
 		N2 interface{}
 	}
 
+	civDateVal := civil.Date{
+		Year:  2020,
+		Month: 11,
+		Day:   10,
+	}
+	civTimeValNano := civil.Time{
+		Hour:       1,
+		Minute:     1,
+		Second:     1,
+		Nanosecond: 1,
+	}
+	civTimeVal := civil.Time{
+		Hour:   1,
+		Minute: 1,
+		Second: 1,
+	}
+	timeValNano, _ := time.Parse("15:04:05.000000000", "01:01:01.000000001")
```

Comment:
Why skip errors?

Reply:
Args are constant, so as the value of the method.